src/config/wmodules-config.c

```diff
@@ -134,18 +134,20 @@ int Read_WModule(const OS_XML *xml, xml_node *node, void *d1, void *d2)
         mwarn("A deprecated Vulnerability Detector configuration block was found. It will be ignored.");
         OS_ClearNode(children);
         return 0;
-    } else if (!strcmp(node->values[0], WM_AZURE_CONTEXT.name)) {
-        if (wm_azure_read(xml, children, cur_wmodule) < 0) {
+    }
+    else if (!strcmp(node->values[0], WM_KEY_REQUEST_CONTEXT.name)) {
+        if (wm_key_request_read(children, cur_wmodule) < 0) {
             OS_ClearNode(children);
             return OS_INVALID;
         }
-    } else if (!strcmp(node->values[0], WM_KEY_REQUEST_CONTEXT.name)) {
-        if (wm_key_request_read(children, cur_wmodule) < 0) {
+    }
+#endif
+else if (!strcmp(node->values[0], WM_AZURE_CONTEXT.name)) {
```

Comment:
Add missing identation here. This should be at the same level as the previous `else if` condition.